### PR TITLE
Creature return and regen after killing player. Also allowing Logger colors in windows.

### PR DESCRIPTION
**Pull request by Chyrri**
Fixed points by @GrenderG . Logger now works in PowerShell, CMD and IDE (atleast VSC). Removed wierdness with UnitManager.py.

### CODE REVIEW

**Comment by Chyrri** [comment]
For issue #234 ** Switched to personal github account.

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,6 @@ venv
 
 # python win libs
 **.pyd
+
+#config.yml
+./etc/config/config.yml
```
Comment (GrenderG) [inline .gitignore:29]: The config.yml file is already being ignored in line 8, why did you add it again?

File: utils/Logger.py [hunk 2]
```diff
@@ -37,9 +36,7 @@ def _should_log(log_type: DebugLevel):
     @staticmethod
     def _colorize_message(label, color, msg):
         date = datetime.now().strftime('[%d/%m/%Y %H:%M:%S]')
-        # No colors for Windows :)
-        if Logger.IS_WINDOWS:
-            return f'{label} {date} {msg}'
+        # Colorama seems to be working with windows nowadays.
```
Comment (GrenderG) [inline utils/Logger.py:39]: Did you test with both the `cmd` and PowerShell? Also in the embedded terminal of your IDE.

**Comment by GrenderG** [comment]
Something is weird about [the commit](https://github.com/The-Alpha-Project/alpha-core/pull/264/commits/4e1eea4063456e9c11b098bcecea83cfd1536173) @Chyrri, it appears as if you added the entire UnitManager file from scratch. Do you know what happened?

File: .gitignore (hunk 1 above)
Comment (Chyrri) [reply .gitignore:29]: That is a good question! Removing.

File: utils/Logger.py [hunk 3]
```diff
@@ -27,7 +28,10 @@ class DebugLevel(IntEnum):
 
 
 class Logger:
-    IS_WINDOWS = platform == 'win32'
+    
+    # If windows override whether to convert ANSI codes in the output into win32 calls.
+    if platform == 'win32':
+        init(convert=True)
```
Comment (GrenderG) [inline utils/Logger.py:34]: Can you please move this to line 30 of main.py? `colorama.init()` is being called there too. Maybe send `convert=platform == 'win32` or `convert=platform == 'nt'`, maybe the later is more correct in fact, not sure.

Comment (GrenderG) [reply utils/Logger.py:34]: Also please confirm that it works by testing in cmd, PowerShell and the integrated IDE console, I don't have a Windows machine ready to test it.

File: .gitignore [hunk 4]
```diff
@@ -23,4 +23,4 @@ venv
 **.idx
 
 # python win libs
-**.pyd
+**.pyd
```
Comment (GrenderG) [inline .gitignore:26]: You removed the new line at the end. :P

File: utils/Logger.py (hunk 3 above)
Comment (Chyrri) [reply utils/Logger.py:34]: Apparentally convert isn't necessary. Removing init() from Logger.py causes colors to work only when Logger methods are called from main.py where init() is called aswell. Calling init() (w/o convert) makes all Logger methods work. Same thing with all three, cmd, PowerShell and integrated IDE of VSC. Doing another commit on this.

**Review by Chyrri — COMMENTED** [review]
@GrenderG 
coloramaWOconvert
^CMD without init() in Logger.py
coloramaWconvert
^ CMD with init() in Logger.py
Works same way in PowerShell and VSC integrated terminal aswell.
